test/e2e/run_test.go

```diff
@@ -48,6 +48,28 @@ var _ = Describe("Podman run", func() {
 		Expect(session).Should(Exit(0))
 	})
 
+	It("podman generates different names for successive containers", func() {
+		SkipIfRemote("The -l,--latest option not supported with remote")
+		var names [2]string
+
+		for i := 0; i < 2; i++ {
```

Comment:
maybe a bit better golang style would be to use
```suggestion
		for i := range names {
```